### PR TITLE
Change Inquirer/prompts dependency to prompts

### PR DESCRIPTION
**Pull request by omkarshanbhag**
## Summary & Motivation

This PR changes it so that we use the `prompts` library instead of `inquirer/prompts` for consistency with our stackblitz examples -- since it seems friendlier to be used with `pnpm` or `npm`

## How I Tested These Changes

Tested the edited examples locally

## Did you add a changeset?

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

File: examples/with-solana/src/addSignature.ts [hunk 1]
```diff
@@ -56,22 +56,22 @@ async function main() {
...
+    await prompts({ message: "Ready to Continue?" });
     // refresh balance...
     balance = await solanaNetwork.balance(connection, solAddress);
   }
 
   print("SOL balance:", `${balance} Lamports`);
 
   // 1. Create, sign, and verify a transfer transaction using `addSignature` as opposed to `signTransaction`
-  const destination = await input({
+  const destination = await prompts({
```
Comment (andrewkmin) [inline examples/with-solana/src/addSignature.ts:67]: was this tested? there are some syntax differences with using `prompts` vs. `inquirer/prompts`. you can refer to the stackblitz example itself https://stackblitz.com/edit/stackblitz-starters-xeb93i?file=src%2Findex.ts

File: examples/with-solana/src/addSignature.ts [hunk 2]
```diff
@@ -56,39 +56,59 @@ async function main() {
...
-    default: TURNKEY_WAR_CHEST,
-  });
+  const { destination } = await prompts([
+    {
+      type: 'text',
+      name: 'destionation',
+      message: `Destination address:`,
+      default: TURNKEY_WAR_CHEST,
+    }
+    ]);
```
Comment (omkarshanbhag) [inline examples/with-solana/src/addSignature.ts:86]: prettier (probably)

File: examples/export-in-node/src/index.ts [hunk 3]
```diff
@@ -1,10 +1,13 @@
 import * as dotenv from "dotenv";
 import * as path from "path";
-import { input } from "@inquirer/prompts";
+import prompts from "prompts";
 import { Turnkey } from "@turnkey/sdk-server";
 import { Crypto } from "@peculiar/webcrypto";
 import { generateP256KeyPair, decryptExportBundle } from "@turnkey/crypto";
-global.crypto = new Crypto();
+if (typeof crypto === "undefined") {
```
Comment (omkarshanbhag) [inline examples/export-in-node/src/index.ts:7]: @andrewkmin @moe-dev Tagging for context

This was the fix for the following error that I ran into while running this example: 

`TypeError: Cannot set property crypto of #<Object> which has only a getter`

Comment (omkarshanbhag) [reply examples/export-in-node/src/index.ts:7]: Same for `import-in-node`

File: examples/with-solana/src/advanced.ts [hunk 4]
```diff
@@ -58,48 +58,69 @@ async function main() {
...
+      type: "text",
+      name: "numTxsStr",
       message: `Number of transactions:`,
-      default: "1",
-    })
-  );
+      initial: "1",
+    },
+  ]);
+  console.log(numTxsStr);
```
Comment (andrewkmin) [inline examples/with-solana/src/advanced.ts:82]: 🧹

**Review by andrewkmin — COMMENTED** [review]
nice! this is super sick. great to consolidate on a single interface for using prompts in our examples, and importantly, makes all of these examples more compatible with stackbiltz/other environments 👏 

I'd like to get to the bottom of the `global.crypto` issue before merging as it's one that I hadn't run into when running locally myself. once resolved, let's 🚢 !

File: examples/export-in-node/src/index.ts (hunk 3 above)
Comment (omkarshanbhag) [reply examples/export-in-node/src/index.ts:7]: Node version --> `v22.5.1`

Pnpm version --> `8.4.0`

File: examples/with-solana/src/advanced.ts (hunk 4 above)
Comment (omkarshanbhag) [reply examples/with-solana/src/advanced.ts]: Done!

**Review by andrewkmin — APPROVED** [review]

File: examples/export-in-node/src/index.ts (hunk 3 above)
Comment (omkarshanbhag) [reply examples/export-in-node/src/index.ts:7]: Posting quick follow up for visibility --> We tested this issue and it seems like this is a node versioning issue 

Old code works with node 18 but not with node 20

**Comment by omkarshanbhag** [comment]
Tested import and export successfully!

**Pull request merged**